org.d3cod3.Mosaic.yml

```diff
@@ -33,7 +35,6 @@ modules:
   - shared-modules/glew/glew.json
   - shared-modules/gtk2/gtk2.json
   - shared-modules/libusb/libusb.json
-  - shared-modules/linux-audio/jack2.json
```

Comment:
this will simply cause the jack headers to not be found, so there won't be any jack support (if it's optional).

The proper way is to add this to the top-level `cleanup`:

```yaml
 - /lib/jack
 - /lib/libjack*
```

Reply:
It is not clear to me, should do I maintain the `- shared-modules/linux-audio/jack2.json` line and add the cleanup, or just the cleanup is enough? I tried to document myself, but there don't seem to be a lot of info available

Reply:
you need both.

It's not well documented, sadly.

Reply:
And I confirm, without JACK, the build fail.

Reply:
Thanks! gonna try now